crawl-ref/source/dat/strings/de/messages.txt

```diff
@@ -10373,10 +10373,10 @@ The frozen ramparts melt away as you move.
 Die gefrorenen Wälle schmelzen, als du dich bewegst.
 %%%%
 You slip free of %s engulfing you.
-Du befreist dich von {dat}%s, das dich verschlingt.
+Du befreist dich von {dat}%s, das dich umhüllt.
```

Comment:
The parameter here will either be "the water"/"das Wasser" or "the ooze"/"der Schlamm" (should probably be "der Schleim"), so I think we have a problem with the relative pronoun "das".